src/components/DataTable/DataTable.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { DataTable, DataTableEmpty, DataTableHeader, DataTableRow } from "."
+
+export default {
+  title: "Components/Data Table",
+}
+
+const Template = {
+  children: "Data Table",
+}
+
+export const DataTableEmptyStory = () => (
+  <DataTable>
+    <DataTableEmpty colspan={0} />
```

Comment:
```suggestion
    <DataTableEmpty />
```

Reply:
Please add separate story with example of empty table that has three cols (`<th>Col 1</th><th>Col 2</th><th>Col 3</th>`)